lib/jekyll/convertible.rb

```diff
@@ -46,7 +46,7 @@ def read_yaml(base, name, opts = {})
           self.content = $POSTMATCH
           self.data = SafeYAML.load(Regexp.last_match(1))
         end
-      rescue SyntaxError => e
+      rescue Psych::SyntaxError => e
```

Comment:
This is very old code – can you leave the old `SyntaxError` and just add the `Psych::SyntaxError`?